src/include/job.h

```diff
@@ -496,9 +496,10 @@ struct jbdscrd {
 
 #define	JSVERSION_514	514
 #define	JSVERSION_80	800
-#define	JSVERSION	800
-#define ji_taskid  ji_extended.ji_ext.ji_taskidx
-#define ji_nodeid  ji_extended.ji_ext.ji_nodeidx
+#define	JSVERSION_18	800
+#define	JSVERSION	900
```

Comment:
It is better to clean-up the macros JSVERSION_514, JSVERSION_80 and its corresponding usage in the code as they are used for older version/s of PBS.

Reply:
Thanks for the review @suresh-thelkar, i think we can't change the value 800 because our
 older versions are using the same value and could be directly affect the overlay upgrade because we are parsing the jobs on the basis of JSVERSION_18(800).

Reply:
@Bhagat-Rajput, the value 800 meant for older release of PBS, since the code clean-up is happening now, we should use a number for the macro that goes into its respective structure which represents the release number for example we can use a value 1800 for JSVERSION_18. This is more readable than using 800.

Reply:
As discussed offline we now write 1900 as a job structure version for the current release or 19.x.